OpenRA.Mods.Common/Traits/World/CreateMPPlayers.cs

```diff
@@ -25,16 +26,24 @@ public void CreatePlayers(World w)
 		{
 			var players = new MapPlayers(w.Map.PlayerDefinitions).Players;
 			var worldPlayers = new List<Player>();
+			var ownerAdded = false;
 
 			// Create the unplayable map players -- neutral, shellmap, scripted, etc.
 			foreach (var kv in players.Where(p => !p.Value.Playable))
 			{
 				var player = new Player(w, null, kv.Value);
 				worldPlayers.Add(player);
+
 				if (kv.Value.OwnsWorld)
+				{
+					ownerAdded = true;
 					w.SetWorldOwner(player);
+				}
 			}
 
+			if (!ownerAdded)
+				throw new InvalidOperationException("Map {0} does not define a player actor owning the world.".F(w.Map.Title));
```

Comment:
I don't know if it has to crash, didn't before. Ideally there would be error message (e.g. on map selection, before start,..) to inform player, crashing to desktop while loading the game looks bad.